torch/_inductor/runtime/benchmarking.py

```diff
@@ -203,6 +222,59 @@ def benchmark_many_gpu(
             self.benchmark_gpu(_callable, *args, **kwargs) for _callable in callables
         ]
 
+    @time_and_count
+    def lazy_benchmark(
+        self: Self,
+        fn: Callable[..., Any],
+        fn_args: Tuple[Any],
+        fn_kwargs: Dict[str, Any],
+        **kwargs: Any,
```

Comment:
it's confusing to me that we have both fn_kwargs and kwargs.. also, i think triton's do_bench api where all of the args, and kwargs were already bound was really nice. any reason to diverge ? (maybe this is an issue from a previous pr)